services/mastodon/mastodon-follow.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService, NotFound } = require('..')
+const { metric } = require('../text-formatters')
+const { optionalUrl, nonNegativeInteger } = require('../validators')
+
+const schema = Joi.object({
+  username: Joi.string().required(),
+  followers_count: nonNegativeInteger,
+})
+
+const queryParamSchema = Joi.object({
+  domain: optionalUrl,
+}).required()
+
+module.exports = class MastodonFollow extends BaseJsonService {
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'mastodon/follow',
+      pattern: ':id',
+      queryParamSchema,
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Mastodon Follow',
+        namedParams: {
+          id: '26471',
+        },
+        queryParams: { domain: 'https://mastodon.social' },
+        staticPreview: {
+          label: 'Follow',
+          message: '862',
+          style: 'social',
+        },
+      },
```

Comment:
I think the only thing left is to add documentation to the examples here. I'd suggest listing your lookup tool as the first option, and then also list the other two (the lookup process and the html tag) as fallback options folks can use.